A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_AirportsMonitor.js

```diff
@@ -212,26 +212,23 @@ class CDUAirportsMonitor {
 
         // user-selected 5th airport (only possible to set on page 1)
         if (!this.page2) {
-            mcdu.onLeftInput[4] = () => {
+            mcdu.onLeftInput[4] = (value) => {
                 if (this.user_ap) {
-                    if (mcdu.inOut === FMCMainDisplay.clrValue) {
+                    if (value === FMCMainDisplay.clrValue) {
                         this.user_ap = undefined;
-                        mcdu.clearUserInput();
                         // trigger data update next frame
                         this.total_delta_t = update_ival_ms;
                         this.ShowPage(mcdu);
                     } else {
-                        console.log(`"${mcdu.inOut}"`);
+                        console.log(`"${value}"`);
                         console.log(`"${FMCMainDisplay.clrValue}"`);
```

Comment:
Please remove `console` calls for performance.

Reply:
done.